src/components/CustomerQuota/ItemsSelection/IdentifierLayout/IdentifierSelectionInput.tsx

```diff
@@ -30,7 +31,11 @@ export const IdentifierSelectionInput: FunctionComponent<{
     >
       <DropdownFilterInput
         label={label}
-        placeholder="Select reason"
+        placeholder={`${c13nt(
+          "selectionPlaceholder",
+          undefined,
+          i18nt("identifierSelectionInput", "placeholder")
+        )}`}
```

Comment:
Do we need `c13nt` here? I think the following is sufficient:

```
placeholder={i18nt("identifierSelectionInput", "placeholder")}
```

Reply:
I thought that there might be other dropdown items like Select Nationality, Select Gifts, etc., so Select Reason might not be enough?